webapp/blueprint.py

```diff
@@ -48,7 +54,10 @@ def root():
 @gui.route("/store")
 @gui.route("/support")
 def jaas():
-    return flask.redirect(JAAS_URL)
+    if flask.request.path == "/home":
+        return flask.redirect(JAAS_URL)
+    else:
+        return flask.redirect(urllib.parse.urljoin(JAAS_URL, flask.request.path))
```

Comment:
are there any tests in the codebase around the urls that we care about to where they end up going?

Reply:
There aren't, we should add them though - I'll create a follow-up issue for this